internal/domain/environment/model.go

```diff
@@ -0,0 +1,20 @@
+package environment
+
+import "github.com/flexprice/flexprice/internal/types"
+
+type Environment struct {
+	ID   string          `db:"id" json:"id"`
+	Name string          `db:"name" json:"name"`
+	Type EnvironmentType `db:"type" json:"type"`
+	Slug string          `db:"slug" json:"slug"`
+
+	types.BaseModel
+}
+
+type EnvironmentType string
+
+const (
+	EnvironmentDevelopment EnvironmentType = "development"
+	EnvironmentTesting     EnvironmentType = "testing"
+	EnvironmentProduction  EnvironmentType = "production"
+)
```

Comment:
move these to `internal/types` as we are defining all types there so that types can be used everywhere in the code without having to import domains specifically. This helps in avoiding import cycles in future (hypothetical ex say subscription model needs `invoice_type` and invoice model needs `subscription_type` then in that case it will lead to import cycles.

Reply:
done